anvio/splitter.py

```diff
@@ -243,8 +243,8 @@ def is_hierarchical_clustering_for_bin_OK(self):
     def is_dbs_identical(self, source_db_path, target_db_path):
         """Check whether the two dbs have identical table names"""
 
-        source_db = db.DB(source_db_path, None, ignore_version=True)
-        target_db = db.DB(target_db_path, None, ignore_version=True)
+        source_db = db.DB(source_db_path, None, ignore_version=True, skip_rowid_prepend=True)
+        target_db = db.DB(target_db_path, None, ignore_version=True, skip_rowid_prepend=True)
```

Comment:
With regards to the commit message in 6fec679cba6acae23dcd5e4a583b56546cca7049, So why would the databases occassionally have a `entry_id` in source but not target? I just ask because `merger.py` also reads from a source and writes it to a target.

Reply:
Nothing read during merge to append to target tables include `entry_ids` (they are all view tables). It is only an issue when the table to read from (or write to) requires ROWID prepend to its data on-the-fly.